src/main.jsx

```diff
@@ -1,10 +1,27 @@
 import React from "react";
 import ReactDOM from "react-dom";
+import { IntlProvider } from "react-intl";
 import App from "./App";
+import Wrapper from "./components/Wrapper";
+import Spanish from "./lang/es-US.json";
+import English from "./lang/en-US.json";
+
+const locale = window.navigator.userLanguage || window.navigator.language;
+
+let messages;
+if (locale === "en-US") {
+  messages = English;
+} else {
+  messages = Spanish;
+}
 
 ReactDOM.render(
   <React.StrictMode>
-    <App />
+    <Wrapper>
+      <IntlProvider locale={locale} messages={messages}>
+        <App />
+      </IntlProvider>
```

Comment:
💡 IntlProvider can probably be moved inside the Wrapper along with the locale stuff. Maybe even move the locale stuff to a custom hook, that might be an idea too.

Reply:
I moved locale and IntlProvider into the Wrapper component.  Commit:  575b7f34ca3626bf210cb62f6c69481d6b51b125

I'll try to make the locale into a custom hook after addressing some of the other comments.